recipe/build.sh

```diff
@@ -1,9 +1,16 @@
+SCONFSFLAGS=""
 if [ -n "${GXX}" ]; then
     # SConstruct wants to find 'g++' in name, and conda will have gnu-c++ in name
-    CXX=${GXX}
+    SCONSFLAGS+="CXX=${GXX} "
 fi
-# Tell it the prefix and compiler
-SCONSFLAGS+="PREFIX=$PREFIX CXX=$CXX "
+
+if [[ $OSTYPE == darwin* ]]; then
+    # Darwin
+    SCONSFLAGS+="CXX=${CLANGXX} DYLD_LIBRARY_PATH=$PREFIX/lib "
```

Comment:
What is this CLANGXX thing?

Reply:
conda-build mangles the name to `*-gnu-c++` for CXX but keeps it `*-clang++` for clang for some reason. This breaks the compiler sniffing in SConstruct.

It also defines GXX for GCC and CLANGXX for clang, where the names end with `-g++` and `-clang++` respectively. For GCC I need to use GXX instead of CXX, I just used CLANGXX to be consistent, but maybe that's not the right thing